tutorials/export-mult-raster/index.md

```diff
@@ -0,0 +1,131 @@
+---
+
+[Open In Code Editor](https://code.earthengine.google.com/7097a7dace39dbaa5025575dae46612f)
+
+It is desirable to pull multiple raster images for environmental public health research. Typically the research question will guide the temporal resolution, but in most cases having multiple images over the course of the year is useful to interrogate possible seasonal patterns in exposure-disease relationships.
+
+In this tutorial, we will demonstrate one approach to pre-specifying the season, and then pulling one raster per season per year. For our exposure data, we will use land surface temperature data. We will conduct this analysis for Botswana.
+
+## Step 1: Draw a polygon around Botswana
+
+We can just use the polygon tool in the map window to draw a polygon. This polygon will be called "geometry" and be imported to the top of the program as an object. We can then define this object in our code as "region."
+```
+var region = geometry;
+
+```
+## Step 2: Set up date variables
+
+Let's say we are interested in pulling temperature data for the years 2000, 2001, and 2002. We can store this information in Google Earth Engine as an array:
+
+```
+// specify years -> can extend this code by adding years to the array
+
+var yrarr = ["2000", "2001", "2002"];
+print(yrarr);
+
+```
+Next, we need to define our start and end dates to filter the image collection. We can do this by defining functions. Note that you can adapt the array to store additional start and end dates for additional months (e.g. April, July, September) by adding additional elements to the arrays. We also want to iterate over years for this, so we are writing a function to add the year that we have predefined to the MM-DD strings.
+
+```
+
+// create array of dates for start -> can extend this code by adding "yr + "" to each element
+// in the array
+
+var makeSt = function(yr){
+  var stArr = [yr + "-01-01"]; 
+  
+  // print(stArr) - this is a sanity check to make sure it's doing what we think;
+  return(stArr);
+};
+
+// create array of dates for end
+var makeEd = function(yr){
+   var eArr = [yr + "-12-31"];
+   
+  //  print(eArr) - this is a sanity check to make sure it's doing what we think;
+   return(eArr);
+};
+
+```
+## Step 3: Write function to apply filters and processing data for export
+
+In this step, we are going to perform the following tasks in sequence as part of a function called "GetImage":
+
+1. We first have to tell Google Earth Engine that the dates we have defined above are in fact date variables.
+2. Next, we filter on the start and end dates specified earlier, and the region object we defined by drawing a polygon around Botswana
+3. It's good practice to check how many images we get - we want to just keep one
+4. We take the earlier image by sorting on this element in the data
+5. Our function spits out a singe image in Botswana for the season we specify
+
+```
+
+// Write a function to grab image with data you need
+var GetImage = function(bdt, edt) {
```

Comment:
Functions usually start with lowercase (`getImage()`) in JavaScript.

Reply:
Thanks - is it essential to change (noting that many community users may not be familiar with Javascript function naming conventions?)

Reply:
Yes please. Even if a user only interacts with JavaScript via Earth Engine, Community Tutorials are published in our official documentation alongside other examples, API docs, where should strive for consistency. Perhaps more importantly, capital `GetImage()` has special meaning in JavaScript (an object constructor), which if used improperly, can confuse learners of the language. 

Thanks for your patience!

Reply:
Understood, I will change in the new version. Thanks!